assets/templates/homepage/main-figures-none.tmpl

```diff
@@ -0,0 +1,6 @@
+<div class="col col--lg-29 col--md-29">
+    <article class="tile tile__content margin-top-lg--2 margin-top-md--2 margin-left-md--1 margin-left-lg--0 height-lg--31">
+        <div class="tile__text-description font-weight-700 margin-top--2 margin-bottom--2"> {{ localise "ExperiencingIssues" .Language 1}} </div>
+        <div class="tile__text-description"> {{ localise "RefreshOrVisitTimeseriesTool" .Language 1 | safeHTML }} </div>
```

Comment:
I think the link styling here doesn't match the rest of the page, no underline, underline appears on hover but should change colour, the focus is missing the yellow glow and thick underline. Focus state can be seen here
![Screenshot 2020-10-21 at 14 19 49](https://user-images.githubusercontent.com/47502916/96725527-daba2d80-13a8-11eb-9da0-4512000a1c6b.png)

Where it should look like:
![Screenshot 2020-10-21 at 14 20 04](https://user-images.githubusercontent.com/47502916/96725605-ed346700-13a8-11eb-8ec3-6b2d2c0c79cf.png)

Reply:
Thanks for spotting this. I'd just copied over the pattern we used for the error message on individual tiles. The change I'll make for this should update it on those individual error messages too